lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_l10n_helper.rb

```diff
@@ -116,6 +117,30 @@ def self.generate_strings_file_from_hash(translations:, output_path:)
           end
           File.write(output_path, builder.to_xml)
         end
+
+        # Downloads the export from GlotPress for a given locale and given filters.
+        #
+        # Also post-process the downloaded content to comment out any potential line with an empty (`""`) translation,
+        # to ensure those would fall back to the English/Base locale at runtime instead. These case should theoretically
+        # never happen as GlotPress should not include empty translations in its exports, but we've seen issues in the
+        # past, so better safe than sorry.
+        #
+        # @param [String] project_url The URL to the GlotPress project to export from, e.g. `"https://translate.wordpress.org/projects/apps/ios/dev"`
+        # @param [String] locale The GlotPress locale code to download strings for.
+        # @param [Hash{Symbol=>String}] filters The hash of filters to apply when exporting from GlotPress.
+        #                               Typical examples include `{ status: 'current' }` (the default) or `{ status: 'review' }`.
```

Comment:
This is no longer the default value as far as the helper method is concerned (that default value has since been moved on the action's `ConfigItem` instead)
```suggestion
        #                               Typical examples include `{ status: 'current' }` or `{ status: 'review' }`.
```